airo-camera-toolkit/airo_camera_toolkit/pinhole_operations/unprojection.py

```diff
@@ -136,14 +137,20 @@ def extract_depth_from_depthmap_heuristic(
         (np.ndarray) a 1D array of the depth values for the specified coordinates
     """
 
-    assert mask_size % 2, "only odd sized markers allowed"
-    assert (
-        depth_percentile < 0.25
-    ), "For straight corners, about 75 percent of the region will be background.. Are your sure you want the percentile to be lower?"
+    if mask_size % 2 == 0:
+        raise ValueError("only odd sized markers allowed")
+    if depth_percentile >= 0.25:
+        # TODO: The question in this error message implies that we should not raise an error, but instead log a warning.
```

Comment:
@tlpss Is there any case where we want users to be able to select a depth percentile of more than 25%?